### ISSUE TITLE
Mark down - Email with strikethrough is not underlined

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#37299]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v1.4.44-0
**Reproducible in staging?:** y
**Issue reported by:** Applause - Internal Team
**Issue found when executing PR:** https://github.com/Expensify/App/pull/36773

## Action Performed:
1. Go to staging.new.expensify.com.
2. Go to any chat.
3. Send a message containing email in strikethrough.

## Expected Result:
The email with strikethrough mark down will be underlined.

## Actual Result:
The email with strikethrough mark down is not underlined.
On Android app, before sending the text, the text preview shows underlined text and the underline disappears after sending the text.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/93399543/eb159d05-d63b-4b70-90c8-d4d2d3043bcb

![image](https://github.com/Expensify/App/assets/93399543/488b9e97-2482-4c71-aeec-2619cfbdb01f)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by puneetlath** [comment]
I agree. I don't think it's a bug, but I'll let @isabelastisser and @youssef-lr confirm before closing. 

Removing the deploy blocker label for now.

**Comment by youssef-lr** [comment]
Yeah I'm personally fine with closing this.
